File: rust/src/escape.rs
pub const ERASE_ALL: &[u8] = b"\x1B[2J";
pub const ERASE_LINE: &[u8] = b"\x1B[K";
pub const CURSOR_HOME: &[u8] = b"\x1B[H";

pub const PRIVATE_SHOW_CURSOR: &[u8] = b"\x1B[?25h";
pub const PRIVATE_HIDE_CURSOR: &[u8] = b"\x1B[?25l";
pub const PRIVATE_SAVE_SCREEN: &[u8] = b"\x1B[?47h";
pub const PRIVATE_RESTORE_SCREEN: &[u8] = b"\x1B[?47l";
pub const PRIVATE_SAVE_CURSOR: &[u8] = b"\x1B7";
pub const PRIVATE_RESTORE_CURSOR: &[u8] = b"\x1B8";

File: zig/src/escape.zig
//! ANSI escape + control codes.

pub const erase_all = "\x1b[2J";
pub const erase_line = "\x1b[K";
pub const cursor_home = "\x1b[H";

pub const private = struct {
    pub const show_cursor = "\x1b[?25h";
    pub const hide_cursor = "\x1b[?25l";
    pub const save_screen = "\x1b[?47h";
    pub const restore_screen = "\x1b[?47l";
    pub const save_cursor = "\x1b7";
    pub const restore_cursor = "\x1b8";
};
